firewall.conf

```diff
@@ -15,9 +15,9 @@ iptables -A INPUT -p tcp --dport ssh -j ACCEPT
 # Allow for all traffic from anywhere for the following services
 iptables -A INPUT -p tcp --dport 80 -j ACCEPT          # nginx
 iptables -A INPUT -p tcp --dport 443 -j ACCEPT         # nginx (SSL)
-iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
+#iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
```

Comment:
Well, should we just leave them for when the admin will enable those services? Or just mention in the dashboard that they need to be enabled?